contributing/ORIGIN_TRIALS.md

```diff
@@ -0,0 +1,22 @@
+# Origin trials
+Origin trials enable developers to use an experimental feature in production and provide essential feedback.
+
+The AMP Origin Trial process is inspired by the [Chromium Origin Trial process](https://github.com/GoogleChrome/OriginTrials). 
+
+Traditionally, a feature in experimental mode can be used in development, but cannot be pushed to production. With Origin trials, interested developers can opt-in to a test an experimental feature in production, with the following expectations:
```

Comment:
Nit: Second comma is a splice.

Reply:
As long as we're being pedantic, there also shouldn't be a comma before "but" because it isn't joining an independent clause.